TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/blocks/aurorianfurnace.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "极光熔炉",
+    "icon": "theaurorian:aurorianfurnace",
+    "category": "5blocks",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "item": "theaurorian:aurorianfurnace",
+            "text": "与原版的熔炉相同，但你可以在上方放置孔道来加快熔炼速度！$(br2)速度和孔道最大数量可以在配置文件中被修改。孔道之间也可以被摆放上方块以实现无缝覆盖。"
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
            "text": "与原版的熔炉相同，但你可以在上方放置孔道来加快熔炼速度！$(br2)速度和孔道最大数量可以在配置文件中被修改。"
```
删掉的这一句我不会，需要帮助。